src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/SurveyCasesEndpoint.java

```diff
@@ -0,0 +1,155 @@
+package uk.gov.ons.ssdc.supporttool.endpoint;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import uk.gov.ons.ssdc.supporttool.model.dto.CaseSearchResult;
+import uk.gov.ons.ssdc.supporttool.model.dto.ui.UIRefusalTypeDTO;
+import uk.gov.ons.ssdc.supporttool.model.entity.Survey;
+import uk.gov.ons.ssdc.supporttool.model.entity.UserGroupAuthorisedActivityType;
+import uk.gov.ons.ssdc.supporttool.model.repository.SurveyRepository;
+import uk.gov.ons.ssdc.supporttool.security.UserIdentity;
+import uk.gov.ons.ssdc.supporttool.utility.CaseSearchResultsMapper;
+
+@RestController
+@RequestMapping(value = "/searchInSurvey")
+public class SurveyCasesEndpoint {
```

Comment:
minor thing, but the class name is quite different from the class name. One of them should probably change

Reply:
Assuming you mean the route and the class name? 👍

Reply:
yes route sorry